app/Http/Requests/Pagination/PaginationRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Pagination;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PaginationRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'limit'=>'int',
```

Comment:
Add restrictions on numbers. Now it is possible to transfer negative numbers. For offset too